app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -322,6 +327,9 @@ val MIGRATION_10_11 = object : Migration(10, 11) {
         database.execSQL(
             "ALTER TABLE AppSettings add column show_comment_action_bar_by_default INTEGER NOT NULL default 1",
         )
+        database.execSQL(
+            "ALTER TABLE AppSettings add column use_custom_tabs INTEGER NOT NULL default 1",
```

Comment:
I know we haven't released, but its a pain to break dev setups. Add this as a separate migration.